guides/custom_patient_page.md

```diff
@@ -0,0 +1,43 @@
+Custom patient buttons
+======================
+
+Cliniko offers an additional integration option for applications that handle
+patient data. This allows users to open these applications directly from Cliniko
+and potentially synchronize information on a given patient as needed.
+
+How it works for the user
+-------------------------
+
+Users can add these applications in a settings page under
+Settings > Our clinic > Integrations:
+
+![Configuration
+panel](assets/custom-patient-buttons-config.png?raw=true)
+
+This adds a new button to the Patient Actions bar for each individual patient:
+
+![Patient actions with open
+dropdown](assets/patient-actions-open.png?raw=true)
+
+Clicking on one of those buttons causes the user's browser to send a GET request
+to the corresponding URL, with a query string of `patient_id=<Cliniko ID of
+displayed patient>`. If the user is logged into the other application, they
+should be taken directly to that application's information screen about the
+patient with that Cliniko ID. If they're not logged in, they'll be redirected to
+a login screen before they can see the patient information.
+
+How works for integrators
```

Comment:
`How works for integrators.`

Just missing _it_ or _this_ after 'How':
How (it|this) works for integrators